packages/next/build/index.ts

```diff
@@ -408,9 +412,10 @@ export default async function build(dir: string, conf = null): Promise<void> {
   if (staticPages.size > 0) {
```

Comment:
This branch won't activate when there's no static pages in a build, leaving us without SPR pages being prerendered.

Reply:
Updated